### ISSUE TITLE
Resource leakage in the Webui?

### ISSUE THREAD

**Issue by rbjorklin** [mesosphere/marathon#3408]
I have a habit of leaving my computer on during the night and thus leaves my web browser running. This has on several occasions caused crazy load averages on my laptop when coming back to work. This behavior only happens when I leave one or more tabs open with Marathon running. This happens on both Firefox and Chrome so I wouldn't say it's a browser bug unless both of them have the same one.

**Comment by aldipower** [comment]
Hi @rbjorklin , thanx for reporting this.

Could you tell us which UI version you are using? Because we fixed some related issues in the past. Press `g v` in the UI and you should see the version string.

In the meantime, did you tried to turn it off and on again?
Just kidding.. ;)

**Comment by rbjorklin** [comment]
Running two different versions.
0.14.3 and 0.15.5 according to `g v` I'll leave it running with just the 0.15.5 window tonight and get back to you tomorrow.

[I might have tried that once or twice ;)](https://www.youtube.com/watch?v=rksCTVFtjM4)

**Comment by rbjorklin** [comment]
No troubles this morning with just 0.15.5 (part of 0.15.3 package) so I'm closing this.

**Comment by aldipower** [comment]
Great! Thanx for trying this out.

**Comment by Poltergeist** [comment]
So I have set up also a test environment last night with the current master. I used 40 apps on a single machine with a sleep command with a random length between 1 and 1000 seconds, this ensured that there would be some movement in the time. It ran for about 8 hours. At the end I had a Memory usage which was similar to the start so enormous changes. 
<img width="494" alt="screen shot 2016-03-08 at 08 05 50" src="https://cloud.githubusercontent.com/assets/156010/13597146/15b23a88-e517-11e5-8c69-d8b310739a98.png">
Also I tracked the memory and CPU usage of the chrome task during the first hour using the output of ps.

These are the findings I had after adding them to a spreadsheet and making some charts of the data.

![image 2](https://cloud.githubusercontent.com/assets/156010/13597182/6bb7ec2a-e517-11e5-8269-8b7c95a15cdd.png)
![image 1](https://cloud.githubusercontent.com/assets/156010/13597181/6bb6e988-e517-11e5-9b59-3b5109bb6698.png)
![image](https://cloud.githubusercontent.com/assets/156010/13597180/6bb609b4-e517-11e5-9c6e-b995887ef9a4.png)

The spikes are produced by polling the service every couple of seconds which we should think about. 
Tl;dr: There is no memory leak on the current master and everything is still running smoothly after a couple of hours.

**Comment by JD557** [comment]
I think that there might still be a leak in the Marathon UI.

This is particularly obvious in Firefox: If I keep the Marathon UI open for a while, the browser eventually complains about a possibly crashed script. The UI also starts to get unresponsive until I refresh the tab.
